d3a_api_client/websocket_device.py

```diff
@@ -1,17 +1,11 @@
-import asyncio
 import logging
-import threading
 import traceback
-import websockets
-import json
-from time import time
+
+from d3a_interface.client_connections.websocket_connection import WebsocketMessageReceiver
 from d3a_interface.utils import wait_until_timeout_blocking
-from d3a_api_client.utils import retrieve_jwt_key_from_server
-from d3a_api_client.constants import WEBSOCKET_ERROR_THRESHOLD_SECONDS, WEBSOCKET_MAX_CONNECTION_RETRIES, \
-    WEBSOCKET_WAIT_BEFORE_RETRY_SECONDS
 
 
-class WebsocketMessageReceiver:
+class AssetWebsocketMessageReceiver(WebsocketMessageReceiver):
```

Comment:
I think it is better to name this DeviceWebsocketMessageReceiver, in order to retain the same naming as the file name.